app/controllers/pull_requests_controller.rb

```diff
@@ -0,0 +1,17 @@
+class PullRequestsController < ApplicationController
+  def index
+    @pull_requests = PullRequest.by_organizations([organization.id]).limit(20)
+    @serialized_pull_requests = ActiveModel::Serializer::CollectionSerializer.new(
+      @pull_requests, each_serializer: PullRequestSerializer)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Layout/MultilineMethodCallBraceLayout: Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument.